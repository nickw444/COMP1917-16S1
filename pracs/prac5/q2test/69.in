69
-2 4 -8 -5 -3 6 -3 -3 -5 -5 -9 5 1 7 -4 -2 -3 0 -5 4 4 5 5 7 -3 -10 9 -6 -6 9 4 5 5 1 -8 6 -5 8 -10 5 -9 -6 -1 5 3 -8 10 -3 1 6 8 8 6 -7 -3 10 -8 -1 -7 -4 10 2 3 -5 4 10 6 -4 7
