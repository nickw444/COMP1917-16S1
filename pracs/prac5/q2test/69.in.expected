average of array: 0.405797
